resources/views/cities.blade.php

```diff
@@ -0,0 +1,73 @@
+<x-layout>
+    <div class="max-w-6xl m-auto mt-8">
+        <div class="overflow-x-auto relative">
+        <table class="w-full text-sm text-center">
+            <caption class="hidden">Cities</caption>
+            <thead class="text-xs text-gray-800 uppercase border-b border-gray-300">
+            <tr>
+                <th scope="col" class="py-3 px-6">ID</th>
+                <th scope="col" class="py-3 px-6">City</th>
+                <th scope="col" class="py-3 px-6">Country</th>
+                <th scope="col" class="py-3 px-6">Incoming Flights</th>
+                <th scope="col" class="py-3 px-6">Outgoing Flights</th>
+            </tr>
+            </thead>
+            <tbody>
+                <tr class="bg-white border-b border-gray-100">
+                    <form method="POST" action="cities/create">
```

Comment:
take a look at this. Challenge requires you to specifically make requests using Ajax.